src/Idris/CommandLine.idr

```diff
@@ -402,13 +402,16 @@ versionMsg = "Idris 2, version " ++ show version
 
 export
 usage : String
-usage = versionMsg ++ "\n" ++
-        "Usage: idris2 [options] [input file]\n\n" ++
-        "Available options:\n" ++
-        optsUsage ++
-        "\n" ++
-        "Environment variables:\n" ++
-        envsUsage
+usage = """
+  \{ versionMsg }
+  Usage: idris2 [options] [input file]
+
+  Available options:
+  \{ optsUsage }
+
```

Comment:
This creates an extra newline right?

Reply:
This is a direct translation.  With every change, I tested the output to make sure it was the same.

Reply:
My bad, you are right.  Will fix it.